web/src/ChatBox.js

```diff
@@ -411,17 +411,17 @@ class ChatBox extends React.Component {
   }
 
   render() {
-    const getStoreTitle = (store) => {
-      let title = Setting.getUrlParam("title");
-      if (title === null) {
-        title = (!store?.title) ? this.props.displayName : store.title;
-      }
-      return title;
-    };
+    // const getStoreTitle = (store) => {
+    //   let title = Setting.getUrlParam("title");
+    //   if (title === null) {
+    //     title = (!store?.title) ? this.props.displayName : store.title;
+    //   }
+    //   return title;
+    // };
 
     const getStoreAvatar = (store) => (!store?.avatar) ? Conf.AiAvatar : store.avatar;
 
-    const title = getStoreTitle(this.props.store);
+    const title = this.props.displayName;
```

Comment:
what if model provider is wrong or empty? Do you have a fallback solution for title?